src/RadioGroup/test/RadioGroup.visual.js

```diff
@@ -88,17 +88,33 @@ const tests = [
       },
     ],
   },
+  {
+    describe: 'with content',
+    its: [
+      {
+        it: 'in 2 and 3 radios',
+        radioButtonProps: {
```

Comment:
Good visual test, I recommend adding a selection area:

```suggestion
        props: {
          selectionArea: 'always',
        },
        radioButtonProps: {
```